en/flight_controller/snapdragon_flight_offboard_software.md

```diff
@@ -0,0 +1,39 @@
+# Snapdragon Offboard Software Installation
```

Comment:
As above, something like "Setting up Developer Environment".

You could cross link to https://dev.px4.io/en/setup/dev_env_linux_ubuntu.html#snapdragon-flight too

Reply:
I'm not a fan of cross-linking tutorials. For informative purposes, yes. But there's nothing worse than non-self enclosed tutorials. Before you know it, you're recursively 5 tutorial-layers deep and it gets super messy.

Reply:
You make a good point. I'm not a fan of duplication - with your new docs then https://dev.px4.io/en/setup/dev_env_linux_ubuntu.html#snapdragon-flight has a good chance of diverging.

How about we make sure this new tutorial contains EVERYTHING in https://dev.px4.io/en/setup/dev_env_linux_ubuntu.html#snapdragon-flight and we add a cross link from developer guide to state that because it is required by users the content is in the PX4 user guide?